pkg/sql/plan.go

```diff
@@ -320,7 +320,10 @@ type planNodeSpooled interface {
 var _ planNodeSpooled = &spoolNode{}
 
 type flowInfo struct {
-	typ     planComponentType
+	typ planComponentType
+	// diagram is only populated when instrumentationHelper.shouldSaveDiagrams()
+	// returns true. (Even in that case we've seen a sentry report where it was
+	// nil.)
```

Comment:
nit: if you're going to mention the sentry issue it might be nice to add a link to it. up to you.